docs/general/ad-filtering/create-own-filters.md

```diff
@@ -889,9 +881,11 @@ every time AdGuard encounters a cookie called `NAME` in a request to `example.or
 * `$cookie=/__utm[a-z]/` blocks Google Analytics cookies everywhere
 * `||facebook.com^$third-party,cookie=c_user` prevents Facebook from tracking you even if you are logged in
 
-`$cookie` rules are not affected by regular exception rules (`@@`) unless it is a `$document` exception. In order to disable a `$cookie` rule, the exception rule should also have a `$cookie` modifier. How it works:
+There are two ways to disable `$cookie` rules: first, default way, is to use exception with `@@` mark - `@@||example.org^$cookie`. Another way is to use `$urlblock` exception (which is also contained in the exception alias `$document` - `$elemhide,jsinject,content,urlblock,extension`)
```

Comment:
I'm not sure that first way can be called "default way". comparing to the second one, maybe "obvious" or "direct"?